docs/theming/high-contrast-mode.md

```diff
@@ -0,0 +1,228 @@
+
+<Tabs groupId="framework" defaultValue="angular" values={[{ value: 'angular', label: 'Angular' }, { value: 'javascript', label: 'Javascript' }, { value: 'react', label: 'React' }, { value: 'vue', label: 'Vue' }]}>
+
+<TabItem value="angular">
+
+```css
+@import '@ionic/angular/css/themes/high-contrast.class.css';
+@import '@ionic/angular/css/themes/high-contrast-dark.class.css';
+```
+
+</TabItem>
+<TabItem value="javascript">
+
+```ts
+import '@ionic/core/css/themes/high-contrast.class.css';
+import '@ionic/core/css/themes/high-contrast-dark.class.css';
+```
+
+</TabItem>
+<TabItem value="react">
+
+```tsx
+import '@ionic/react/css/themes/high-contrast.class.css';
+import '@ionic/react/css/themes/high-contrast-dark.class.css';
+```
+
+</TabItem>
+<TabItem value="vue">
+
+```ts
+import '@ionic/vue/css/themes/high-contrast.class.css';
+import '@ionic/vue/css/themes/high-contrast-dark.class.css';
+```
+
+</TabItem>
+
+</Tabs>
+
+This approach activates the high contrast theme when the `.ion-theme-high-contrast` class is set on the `html` element. This class must be applied by the developer. This can be combined with the [`.ion-theme-dark` class](./dark-mode#class) to conditionally apply the high contrast dark theme.
+
+The following example combines site settings, system settings, and the toggle to decide when to show high contrast mode. The site's theme takes precedence over system settings. If your system settings differ from the site's theme when the demo loads, it will use the site's theme.
+
+:::info
+Not sure how to change the system settings? Here's how to enable high contrast mode on [Windows 11](hhttps://support.microsoft.com/en-us/windows/turn-high-contrast-mode-on-or-off-in-windows-909e9d89-a0f9-a3a9-b993-7a6dcee85025) and on [macOS](https://support.apple.com/guide/mac-help/change-display-settings-for-accessibility-unac089/mac).
+:::
+
+import ClassHighContrastMode from '@site/static/usage/v8/theming/class-high-contrast-mode/index.md';
+
+<ClassHighContrastMode />
+:::caution Important The high contrast light theme must be imported after [core.css](../layout/global-stylesheets#corecss),
+and the high contrast dark theme must be imported after `dark.class.css`. Otherwise, the standard contrast theme will take
+priority. :::
+
+:::caution Important
+The `.ion-theme-high-contrast` class **must** be added to the `html` element in order to work with the imported high contrast theme.
+:::
+
+## Customizing Ionic High Contrast Theme
```

Comment:
Should we include a note here about how you need to include the dark files to work with dark mode + apply the dark mode class or have the system settings set for the class and system files, respectively?

Reply:
https://github.com/ionic-team/ionic-docs/pull/3431/commits/daadec0488b5887cc846fee188afa8ec9f9a82cc